InnerEye/ML/dataset/scalar_dataset.py

```diff
@@ -747,13 +784,21 @@ def get_labels_for_imbalanced_sampler(self) -> List[float]:
         """
         return [item.label.item() for item in self.items]
 
-    def get_class_counts(self) -> Dict:
+    def get_class_counts(self) -> Dict[float, int]:
```

Comment:
is the type annotation correct? In the multi-label case, it looks like Dict[int,int] to me.